<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hyperion Gun</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="SliderX">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>590</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>4</height>
     </size>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetting">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>81</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe Gothic Std B</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REMOTO</string>
    </property>
   </widget>
   <widget class="QSlider" name="SliderY">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonArmed">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>250</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>81</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>ARMAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShoot">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>390</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>81</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>DISPARAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInformationServerClient">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>180</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Modo: Sin conexi√≥n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditStringData">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>590</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="EnableSliderButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>81</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe Gothic Std B</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MANUAL</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="DisplayX">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>580</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="DisplayY">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>670</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="LabelInfo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>650</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Usa ASDW para mover los sliders  </string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTime">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>610</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>LabelTime</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelPort">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>610</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>LabelPort</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCapture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>581</width>
      <height>481</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-style: solid;
border-color: yellow;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditCamera">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-style: dashed;
border-color: yellow;
color: white;
background-color: black
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenCloseCamera">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>81</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe Gothic Std B</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AUTOMATICO</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCaptureMask">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>581</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-style: solid;
border-color: yellow;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSerialPort">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>650</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe Gothic Std B</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>811</width>
      <height>191</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: yellow;
border-width: 5px;
border-style: solid;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-10</y>
      <width>1201</width>
      <height>824</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>590</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Gothic Std B</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>570</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: yellow;
border-width: 5px;
border-style: solid;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>SliderX</zorder>
   <zorder>pushButtonSetting</zorder>
   <zorder>SliderY</zorder>
   <zorder>pushButtonArmed</zorder>
   <zorder>pushButtonShoot</zorder>
   <zorder>labelInformationServerClient</zorder>
   <zorder>EditStringData</zorder>
   <zorder>SendButton</zorder>
   <zorder>EnableSliderButton</zorder>
   <zorder>DisplayX</zorder>
   <zorder>DisplayY</zorder>
   <zorder>LabelInfo</zorder>
   <zorder>LabelTime</zorder>
   <zorder>LabelPort</zorder>
   <zorder>LabelCapture</zorder>
   <zorder>plainTextEditCamera</zorder>
   <zorder>pushButtonOpenCloseCamera</zorder>
   <zorder>LabelCaptureMask</zorder>
   <zorder>ButtonSerialPort</zorder>
   <zorder>pushButtonOpenCloseCamera</zorder>
   <zorder>pushButtonSetting</zorder>
   <zorder>EnableSliderButton</zorder>
   <zorder>SendButton</zorder>
   <zorder>LabelTime</zorder>
   <zorder>LabelPort</zorder>
   <zorder>EditStringData</zorder>
   <zorder>SendButton</zorder>
   <zorder>ButtonSerialPort</zorder>
   <zorder>pushButtonShoot</zorder>
   <zorder>pushButtonArmed</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background-color: black</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: black</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
